modules/solutions/twitter-analytics/org.wso2.sp.portal.twitter.analytics.solution.feature/src/main/resources/p2.inf

```diff
@@ -0,0 +1,5 @@
+instructions.configure = \
+org.wso2.carbon.extensions.touchpoint.mkdir(path:${installFolder}/../{runtime}/deployment/);\
+org.wso2.carbon.extensions.touchpoint.mkdir(path:${installFolder}/../{runtime}/deployment/web-ui-apps/);\
+org.wso2.carbon.extensions.touchpoint.mkdir(path:${installFolder}/../{runtime}/deployment/web-ui-apps/monitoring/);\
+org.wso2.carbon.extensions.touchpoint.copy(source:${installFolder}/../lib/features/org.wso2.carbon.status.dashboard.web_${feature.version}/monitoring/,target:${installFolder}/../{runtime}/deployment/web-ui-apps/monitoring/, overwrite:true);\
```

Comment:
The widgets should be copied to /{runtime}/deployment/web-ui-apps/portal/extension/widgets NOT
/{runtime}/deployment/web-ui-apps/monitoring/

Reply:
Your p2 file indicates it's being copied to monitoring 
"org.wso2.carbon.extensions.touchpoint.copy(source:${installFolder}/../lib/features/org.wso2.carbon.status.dashboard.web_${feature.version}/monitoring/,target:${installFolder}/../{runtime}/deployment/web-ui-apps/monitoring/, overwrite:true);"
rather than portal folder.